GetandTake/Models/DTOs/BaseDTO/ProductDTO.cs

```diff
@@ -0,0 +1,19 @@
+﻿using GetandTake.Models.Base;
+
+namespace GetandTake.Models.DTOs.BaseDTO
+{
+    public class ProductDTO : IBaseDTO
+    {
+        //use for Create,Update,Delete
```

Comment:
There is a good practice that code should be self-documented (well, there are some exceptions, but not in our case). Leaving such comment is not necessary and, I would say, redundant in some way. Check out, please, whever you have time about comments best practices [here](https://www.codeguru.com/csharp/commenting-best-practices-in-c/)